spec/lib/catalog/approval_transition_spec.rb

```diff
@@ -34,7 +34,7 @@
         expect(order_item_transition.process.state).to eq "Approved"
       end
 
-      it "calls out to Catalog::SubmitOrder" do
+      it "calls out to Catalog::SubmitOrderItem" do
```

Comment:
I know it's just an it block description, but it should probably read `SubmitNextOrderItem`